pandas/_libs/tslibs/parsing.pyx

```diff
@@ -42,6 +49,99 @@ cdef:
     set _not_datelike_strings = {'a', 'A', 'm', 'M', 'p', 'P', 't', 'T'}
 
 # ----------------------------------------------------------------------
+cdef:
+    const char* delimiters = " /-."
+    int MAX_DAYS_IN_MONTH = 31, MAX_MONTH = 12
+
+
+cdef bint _is_not_delimiter(const char ch):
```

Comment:
can be inline?

Reply:
it should be, thanks for noticing! and thanks @anmyachev for prompt responding.